src/libraries/Microsoft.PowerFx.Core/Types/Enums/BuiltInEnums.cs

```diff
@@ -199,5 +199,16 @@ private static Dictionary<string, object> TraceSeverityDictionary()
                 { "IgnoreUnsupportedTypes", TraceFunction.IgnoreUnsupportedTypesEnumValue },
             },
             canConcatenateStronglyTyped: true);
+
+        public static readonly EnumSymbol JoinTypeEnum = new EnumSymbol(
+            new DName(LanguageConstants.JoinTypeName),
+            DType.String,
+            new Dictionary<string, object>()
+            {
+                { "Inner", "inner" },
```

Comment:
nit - What is the object (value) here? Can we build this around a strongly typed C# enum?